tests/scripts/validate_cluster.sh

```diff
@@ -89,17 +89,20 @@ function test_demo_pool {
 
 function test_csi {
   timeout 360 bash -x <<-'EOF'
+    echo $IS_POD_NETWORK
     echo $IS_MULTUS
-    if [ -z $IS_MULTUS ]; then
+    if [ -z $IS_POD_NETWORK ]; then
       until [[ "$(kubectl -n rook-ceph get pods --field-selector=status.phase=Running|grep -c ^csi-)" -eq 4 ]]; do
         echo "waiting for csi pods to be ready"
         sleep 5
       done
     else
       until [[ "$(kubectl -n rook-ceph get pods --field-selector=status.phase=Running|grep -c ^csi-)" -eq 6 ]]; do
-        echo "waiting for csi pods to be ready with multus"
+        echo "waiting for csi pods to be ready with multus or pod networking"
         sleep 5
       done
+    fi
+    if [ ! -z $IS_MULTUS ]; then
```

Comment:
```suggestion
    if [ -n $IS_MULTUS ]; then
```

Reply:
Use `-n` now?

Reply:
@leseb with `-n` its failing again CI logs here https://github.com/rook/rook/runs/6750483568?check_suite_focus=true. not sure why it's failing